file_optical.c

```diff
@@ -1319,6 +1321,8 @@ static void *fbo_async_format(void *arg)
 	struct fbo_state *state = tcmur_dev_get_private(dev);
 	uint8_t sense[SENSE_BUFFERSIZE];
 
+	tcmu_set_thread_name("fbo-async-format", NULL);
```

Comment:
You can just name this "fbo-format" so it is "handler_name-command" like the fbo-sync-cache one.